libs/features/personalization/personalization.js

```diff
@@ -475,11 +475,17 @@ function cleanManifestList(manifests) {
 }
 
 export async function applyPers(manifests) {
-  if (!manifests?.length) return;
-  const cleanedManifests = cleanManifestList(manifests);
-
   const config = getConfig();
 
+  if (!manifests?.length) {
+    if (config.mep?.preview) {
```

Comment:
Should we check if the query parameter is set? The button also shows without it https://main--homepage--adobecom.hlx.page/?milolibs=mep-button-on-target-only

Reply:
I think this happens here:

https://github.com/adobecom/milo/blob/31810376b5856facde23a3654591383dfe7c1178/libs/features/personalization/add-preview-to-config.js#L10-L14

Seems that on preview pages, the presence of the `mep` query parameter is totally irelevant. You would need to set `mepButton=off` to avoid loading the button: https://main--homepage--adobecom.hlx.page/?milolibs=mep-button-on-target-only&mepButton=off. I also see that it doesn't load on `.live` pages.